spec/models/user_spec.rb

```diff
@@ -38,43 +37,80 @@
   end
 
   describe 'OAuth Methods' do
-
-    def oauth_params
-      {
-        'uid' => '90210',
-        'credentials' => { 'token' => 'averysecrettoken' },
-        'info' => {
-          'name' => 'John Galt',
-          'email' => 'somewhere@colorado.com'
+    describe 'with Census including roles' do
+      def oauth_params(roles: ['student'])
+        {
+          'uid' => '90210',
+          'credentials' => { 'token' => 'averysecrettoken' },
+          'info' => {
+            'name' => 'John Galt',
+            'email' => 'somewhere@colorado.com',
+            'roles' => roles.map { |role| { 'name' => role } }
+          },
+          'provider' => User::AUTH_PROVIDER_GITHUB
```

Comment:
Just for my own clarification, why are we using `User::AUTH_PROVIDER_GITHUB` instead of census?

Reply:
Whoops! I got it wrong and fixed it in cbcfdcc. It's fine either way since we map roles regardless but it's now fixed.